dogs-initial/src/main/java/com/lambdaschool/dogsinitial/Services/MessageListenerHigh.java

```diff
@@ -0,0 +1,21 @@
+package com.lambdaschool.dogsinitial.Services;
+
+import com.lambdaschool.dogsinitial.DogsinitialApplication;
+import com.lambdaschool.dogsinitial.model.MessageDetail;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
+import org.springframework.stereotype.Service;
+
+@Service
+public class MessageListenerHigh
+{
+    private static final Logger logger =LoggerFactory.getLogger(MessageListenerHigh.class);
+
+    @RabbitListener(queues = DogsinitialApplication.QUEUE_NAME_HIGH)
+    public void receiveHighMessage(MessageDetail msg)
+    {
+        //process the message
```

Comment:
Don't forget to also set up a message listener for your other queue - you can do so in a separate file so that you can later on configure the two message queue listeners to be logged to different log files.